lib/floe.rb

```diff
@@ -34,6 +30,8 @@
 require_relative "floe/workflow/states/task"
 require_relative "floe/workflow/states/wait"
 
+require_relative "floe/runner/container_runner"
```

Comment:
cool thing here.

If you extract the gem.
and you put `gem "floe-container_runner", :require => true` into the `Gemfile`, then you can remove this line and ruby will register the plugin without much fuss.

Reply:
Yes exactly.  I actually considered dropping this line entirely and just having exe/floe do the require directly, but didn't want to rock the boat too much.